test/test_base.py

```diff
@@ -312,3 +314,41 @@ def test_provided_templates(self):
                     msg="Should not fail on %s(%r) " % (label, tpl) +
                     "Current stderr:\n%s" % indent(err))
 
+    def test_run_packages_not_in_config(self):
+        out, err, errlvl = cmd('$tprog --package app')
+        self.assertContains(
+            err.lower(), "missing value",
+            msg="There should be an error message containing 'missing value'. "
+            "Current stderr:\n%s" % err)
```

Comment:
`%` aaaaaaaaaaaaa! :)

Reply:
:) c/p from one of the older test. Will adjust as f-string

Reply:
Updated